internal/build/build.go

```diff
@@ -83,20 +86,22 @@ func (l *Lifecycle) Execute(ctx context.Context, opts LifecycleOptions) error {
 	l.Setup(opts)
 	defer l.Cleanup()
 
+	phaseFactory := NewDefaultPhaseFactory(l)
+
 	buildCache := cache.NewVolumeCache(opts.Image, "build", l.docker)
-	launchCache := cache.NewVolumeCache(opts.Image, "launch", l.docker)
 	l.logger.Debugf("Using build cache volume %s", style.Symbol(buildCache.Name()))
-
 	if opts.ClearCache {
 		if err := buildCache.Clear(ctx); err != nil {
 			return errors.Wrap(err, "clearing build cache")
 		}
 		l.logger.Debugf("Build cache %s cleared", style.Symbol(buildCache.Name()))
 	}
 
-	phaseFactory := NewDefaultPhaseFactory(l)
+	launchCache := cache.NewVolumeCache(opts.Image, "launch", l.docker)
 
-	if semver.MustParse(l.platformAPIVersion).LessThan(semver.MustParse("0.3")) {
+	// Technically the creator is supported as of platform API version 0.3 (lifecycle version 0.7.0+) but earlier versions
```

Comment:
See comment here: https://github.com/buildpacks/pack/pull/631/files#r425953889 - maybe we can just use `opts.UseCreator` here.